pycmds/_record.py

```diff
@@ -710,24 +443,24 @@ def initialize_scan(self, aqn, scan_folder, destinations_list):
         headers.pycmds_info["PyCMDS version"] = g.version.read()
         headers.pycmds_info["system name"] = g.system_name.read()
         headers.pycmds_info["file created"] = timestamp.RFC3339
-        # apply device settings from aqn
-        ms_wait.write(aqn.read("device settings", "ms wait"))
-        for device in self.devices:
-            if not aqn.has_section(device.name):
-                device.active = False
-                continue
-            if not aqn.read(device.name, "use"):
-                device.active = False
-                continue
-            # apply settings from aqn to device
-            device.active = True
-            device.load_settings(aqn)
-            # record device axes, if applicable
-            if device.has_map:
-                for key in device.map_axes.keys():
+        # apply sensor settings from aqn
+        ms_wait.write(aqn.read("sensor settings", "ms wait"))
+        for sensor in self.sensors:
+            # if not aqn.has_section(sensor.name):
```

Comment:
is the `use` functionality intentionally omitted?

Reply:
not really, just taken out for now

Reply:
I'm confused by this statement

Reply:
ping @untzag ... wish you would have responded to this before merging...